app/loggers/log_worker_queue_stats.rb

```diff
@@ -0,0 +1,30 @@
+class LogWorkerQueueStats
```

Comment:
what do you think about moving this to `app/labor` ? Most of the "doer" tasks that are called by the scheduler sit there

Reply:
The reason I made a loggers folder is that I also started [a branch with a Sidekiq Tracker](https://github.com/thepracticaldev/dev.to/compare/mstruve/sidekiq-logging) for logging around the workers and I figured all of that could go in that folder. 

I guess `app/labor/loggers` works too, I just want to keep them all together and easy to find.